avm/ptn/authorization/pim-role-assignment/modules/management-group.bicep

```diff
@@ -76,7 +76,7 @@ resource pimEligibleRoleAssignment 'Microsoft.Authorization/roleEligibilitySched
     roleDefinitionId: roleDefinitionIdVar
     requestType: requestType
     condition: condition
-    conditionVersion: conditionVersion
+    conditionVersion: !empty(condition) ? conditionVersion : null
```

Comment:
Are you sure this also required (aside from 'just' setting the `condition` to `'''` (if null does not seem to be supported)